autoware_launch/launch/autoware.launch.xml

```diff
@@ -65,8 +66,9 @@
 
   <!-- System -->
   <group if="$(var launch_system)">
-    <include file="$(find-pkg-share system_launch)/launch/system.launch.xml">
+    <include file="$(find-pkg-share autoware_launch)/launch/components/tier4_system_component.launch.xml">
       <arg name="run_mode" value="$(var system_run_mode)"/>
+      <arg name="launch_system_monitor" value="$(var launch_system_monitor)"/>
       <arg name="sensor_model" value="$(var sensor_model)"/>
```

Comment:
nits: better to remove all the arguments here like this change https://github.com/autowarefoundation/autoware_launch/pull/185/files.

This modification was adopted based on the TIER IR's internal discussion.
https://star4.slack.com/archives/C4P0NSMB5/p1674616811421119

Reply:
For me, it's okay to deal with this comment with another PR once this PR is merged.

Reply:
OK, I will handle it in another PR.